ml4gw/waveforms/phenom_p.py

```diff
@@ -412,25 +418,25 @@ def L2PNR(self, v: TensorType, eta: TensorType) -> TensorType:
 
     def convert_spins(
         self,
-        m1: TensorType,
-        m2: TensorType,
+        m1: Float[Tensor, ""],
+        m2: Float[Tensor, ""],
         f_ref: float,
-        phiRef: TensorType,
-        incl: TensorType,
-        s1x: TensorType,
-        s1y: TensorType,
-        s1z: TensorType,
-        s2x: TensorType,
-        s2y: TensorType,
-        s2z: TensorType,
+        phiRef: Float[Tensor, ""],
```

Comment:
Would this be the same as the `ScalarTensor` in `ml4gw.types` ?

Reply:
Yeah, good catch. Also, some of the types in the waveforms files are definitely wrong - I must've done an overzealous find and replace. Let me get that fixed up.